src/io/flutter/run/daemon/FlutterApp.java

```diff
@@ -597,6 +597,10 @@ public String deviceId() {
     return myDevice != null ? myDevice.deviceId() : "";
   }
 
+  public boolean isWebDev() {
+    return myDevice == null;
```

Comment:
nit: add a TODO that this is a temp hack.

Reply:
TODO added